### PR TITLE
Add descript.app

### PR DESCRIPTION
**Pull request by freeslugs**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by freeslugs** [comment]
Installation and uninstallation work with a few caveats would love to get some help on:
1. I'm using `export HOMEBREW_FORCE_BREWED_CURL=1` to avoid a curl SSL error 
2. The installation opens the app and prints the logs of the app into the terminal -> should we pipe the output to dev/null? 
3. Uninstall uses `delete`, as per their docs, just remove the app from applications folder.

**Comment by freeslugs** [comment]
And they don't use versioning on the download page.

**Comment by freeslugs** [comment]
Any ideas why the CI timed out?

File: Casks/descript.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask "descript" do
+  version :latest
+  sha256 :no_check
+
+  url "https://electron.descript.com/Descript%20Installer.app.zip"
+  name "Descript"
+  homepage "https://www.descript.com/"
+
+  installer script: "Descript Installer.app/Contents/MacOS/Descript Installer"
```
Comment (vitorgalvao) [inline Casks/descript.rb:9]: This needs to use [`installer :manual`](https://github.com/homebrew/homebrew-cask/blob/c129758c8086d86b3d33cc8c2045b4a26bee2a83/doc/cask_language_reference/stanzas/installer.md#installer-manual).

File: Casks/descript.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+  # uninstall pkgutil: 'com.descript.beachcube' # this didnt remove the app
```
Comment (vitorgalvao) [inline Casks/descript.rb:11]: Remove. This is not a `pkg`.

File: Casks/descript.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  uninstall delete: "/Applications/Descript.app"
```
Comment (vitorgalvao) [inline Casks/descript.rb:12]: > Uninstall uses delete, as per their docs

Can you link me?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Just tried it on a VM and this is one of those bad “installers” which needs to be opened manually and just downloads the app which is *1.5GB*.

Thank you for the submission, but we won’t accept this one, they’re an awful user experience that we can do nothing to improve.

**Pull request closed without merging**